.github/workflows/test-etcd.yml

```diff
@@ -0,0 +1,39 @@
+on:
+    push:
+        branches:
+            - master
+            - main
+        paths:
+            - 'etcd/**'
+    pull_request:
+        paths:
+            - 'etcd/**'
+name: "Tests Etcd"
+jobs:
+    Tests:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                go-version:
+                    - 1.19.x
+                    - 1.20.x
+        steps:
+            -   name: Fetch Repository
+                uses: actions/checkout@v3
+            -   name: Install etcd
+                run: |
+                    docker pull bitnami/etcd:latest
```

Comment:
This pull is not needed, the docker run will do it for you

Reply:
done https://github.com/gofiber/storage/pull/878/commits/c592c8b5493e74e819dc184be9efc4be3f259da8 .